staff/asix/full-task-app-1.0.0/src/axios.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios')
+const baseUrl = 'http://localhost:5000/api/tasks'
+
+const axiosCall = {
+
+    list(res){
+        axios.get(`${baseUrl}`)
+        .then(obj => res.render('form', {tasks: obj.data}))
+    },
+
+    create(res, text){
+        axios.post(`${baseUrl}`, { text: text })
+        res.redirect('/')
+    },
+
+    edit(res, id){
+        axios.put(`${baseUrl}/${id}`)
+        res.redirect('/')
+    },
+
+    delete(res, id){
```

Comment:
As for responsability purposes I would include render on index.js;

`list(res {
return axios.get(`${baseUrl}`)
}
`